### PR TITLE
task components part1

### PR DESCRIPTION
**Pull request by JuliaTL**

### CODE REVIEW

File: src/components/CardList/index.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            "budget": 175000000,
+            "revenue": 0,
+            "genres": [
+                "Adventure",
+                "Science Fiction",
+                "Action"
+            ],
+            "runtime": 140
+        },
+    ]
```
Comment (lotrien) [inline src/components/CardList/index.js:116]: do we really need this loong list inside the component? or we can move it somewhere else? What do you think?

File: src/components/Dashboard/Filter/filter.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.breadcrumb {
+  display: flex;
+  justify-content: space-between;
+  padding: 1rem;
+  list-style: none;
+  background-color: #232323;
```
Comment (lotrien) [inline src/components/Dashboard/Filter/filter.scss:6]: I saw this color hex previously. You might consider creating scss variables and use them

File: src/components/Layout/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Header } from '../Header';
+import { Footer } from "../Footer";
+import "../../configs";
```
Comment (lotrien) [inline src/components/Layout/index.js:4]: it's really confusing to name `styles` -> `configs`. Styles are not configs by theirs nature.

File: src/components/Dashboard/Filter/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import './filter.scss';
+
+const Filter = () => {
+        const options = ['Release Date', 'Rating', 'Genres'];
+        return (
+            <>
+                <ul className="breadcrumb">
```
Comment (nullpr) [inline src/components/Dashboard/Filter/index.js:8]: Maybe it would be better to put items into an array, and use map to generate li items?

**Comment by lotrien** [comment]
hey @JuliaTL before you merge it, I would highly recommend you to reed these few articles:
- https://chris.beams.io/posts/git-commit/ (about how crucial commit messages can be, it might be not so essential in scope of this application rather "nice to have", BUT it could help you and your teammates with your further projects)
- https://levelup.gitconnected.com/how-to-squash-git-commits-9a095c1bc1fc (essential knowledge on how to keep your git history neat). I prefer to have small amount of meaningful commits with good commit messages before merging it.

Apart from this, I've noticed that you tried to use `index` files, but it seems to me in a wrong way: `index` file in component folder should contain only imports and exports and nothing else. So when we trying to import something that we need from some particular component we do not import it from different files but from one (which is `index`). 
For example:
image
You might even go further and create something like this: 
image
image
and eventually import it like: `import { HomePage, NotFound, MovieDetailsPage } from 'pages';`
I would highly recommend sticking to the things I described above, but it's up to you

File: src/components/CardList/index.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Card from "./Card";
+import "./cardlist.scss";
+import PropTypes from 'prop-types';
```
Comment (lotrien) [inline src/components/CardList/index.js:4]: it would be good to have some common style for quotes within project

**Pull request merged**